src/RA_Dlg_AchEditor.cpp

```diff
@@ -83,10 +83,15 @@ INT_PTR CALLBACK AchProgressProc(HWND hDlg, UINT nMsg, WPARAM wParam, _UNUSED LP
 Dlg_AchievementEditor::Dlg_AchievementEditor() noexcept
 {
     m_lbxGroupNames.front() = _T("Core");
-    for (auto it = std::next(m_lbxGroupNames.begin()); it != m_lbxGroupNames.end(); ++it)
+
+    gsl::index i = -1;
+    for (auto& name : m_lbxGroupNames)
     {
-        const auto i = std::distance(std::next(m_lbxGroupNames.begin()), it);
-        *it = ra::StringPrintf(_T("Alt %02t"), i);
+        i++;
+        if (i == 0)
+            continue;
+        else
+            name = ra::StringPrintf(_T("Alt %02t"), i);
```

Comment:
This was giving me a weird warning. Something about the offset being negative which didn't make any sense. So I did this way instead.